.changeset/new-timers-warn.md

```diff
@@ -0,0 +1,11 @@
+---
+'@aws-amplify/integration-tests': minor
+'@aws-amplify/backend-deployer': minor
+'create-amplify': minor
+'@aws-amplify/plugin-types': minor
+'@aws-amplify/cli-core': minor
+'@aws-amplify/sandbox': minor
+'@aws-amplify/backend-cli': minor
```

Comment:
are all of the versions minor?

Reply:
given where we are today (0.x versions and added feature with api breaks) yes that seems ok.